x/service/relay.go

```diff
@@ -1,69 +1,77 @@
 package service
 
 import (
-	"bytes"
-	"errors"
-	"github.com/pokt-network/pocket-core/types"
-	"io/ioutil"
-	"net/http"
+	"encoding/hex"
+	"github.com/pokt-network/pocket-core/x/blockchain"
 )
 
+// a read / write API request from a hosted (non native) blockchain
 type Relay struct {
-	Blockchain       ServiceBlockchain `json:"blockchain"`
-	Payload          ServicePayload    `json:"payload"`
-	ServiceToken     ServiceToken      `json:"token"`
-	IncrementCounter IncrementCounter  `json:"incrementCounter"`
+	Blockchain         ServiceBlockchain  `json:"blockchain"`       // the non-native blockchain needed to service
+	Payload            ServicePayload     `json:"payload"`          // the data payload of the request
+	ServiceCertificate ServiceCertificate `json:"incrementCounter"` // the authentication scheme needed for work
```

Comment:
Why call it `ServiceCertificate` if it's an increment counter?

Reply:
It's not an increment counter, it contains an increment counter. The service certificate is proof of work completed on both sides. It has a lot more than just the integer